crates/utils/src/lib.rs

```diff
@@ -3,6 +3,7 @@ use cfg_if::cfg_if;
 cfg_if! {
   if #[cfg(feature = "full")] {
     pub mod cache_header;
+    #[allow(clippy::as_conversions)]
     pub mod rate_limit;
```

Comment:
Tried to get rid of these but it got very complicated with Result added in lots of places. Anyway this code should be moved to a separate crate (https://github.com/LemmyNet/lemmy/issues/5550).